p8e-api/src/main/kotlin/io/provenance/engine/config/Properties.kt

```diff
@@ -56,6 +56,7 @@ class ChaincodeProperties {
     @NotNull var txBatchSize: Int = 25
     @NotNull var gasMultiplier: Double = 1.0
     @NotNull var maxGasMultiplierPerDay: Int = 1000
+    @NotNull var blockHeightTimeoutInterval: Int = 12
```

Comment:
curious why `12` as the magic number?

Reply:
It is a minute at an approximate 5 second block cut time, but really somewhat arbitrary